io.conduktor.Conduktor.yaml

```diff
@@ -0,0 +1,79 @@
+app-id: io.conduktor.Conduktor
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/conduktor.sh
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=pulseaudio
+
+  # Legacy X11 support
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+
+  # Notifications and such
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+
+  # Developer tool, thus we can be more generous
+  - --filesystem=home
+  - --socket=ssh-auth
+  
+  # Java paths
+  - --env=PATH=/app/clients/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - "/usr/lib/sdk/openjdk11/install.sh"
+
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 io.conduktor.Conduktor.desktop /app/share/applications/io.conduktor.Conduktor.desktop
+      - install -Dm644 io.conduktor.Conduktor.metainfo.xml /app/share/metainfo/io.conduktor.Conduktor.metainfo.xml
+      - install -Dm644 io.conduktor.Conduktor.128.png /app/share/icons/hicolor/128x128/apps/io.conduktor.Conduktor.png
+      - install -Dm644 io.conduktor.Conduktor.256.png /app/share/icons/hicolor/256x256/apps/io.conduktor.Conduktor.png
+    sources:
+      - type: file
+        path: io.conduktor.Conduktor.128.png
+      - type: file
+        path: io.conduktor.Conduktor.256.png
+      - type: file
+        path: io.conduktor.Conduktor.desktop
+      - type: file
+        path: io.conduktor.Conduktor.metainfo.xml
+
+  # Download conduktor using an extra-data script
+  - name: conduktor
+    buildsystem: simple
+    build-commands:
+      - install -Dm0755 conduktor.sh /app/bin/conduktor.sh
+      - install -Dm0755 apply_extra /app/bin/apply_extra
+    sources:
+      - type: extra-data
+        filename: conduktor.zip
+        only-arches:
+          - x86_64
+        sha256: 57d6340b7128f5b069db08d50fafb24e70cb97b1dbb1ef3edba5b8fff4d9a95d
+        size: 190604564
+        url: https://releases.conduktor.io/linux-zip
```

Comment:
Would be nice to have x-data-checker here as this link will break eventually.

Reply:
Totally agree. Sadly, I've been trying to get something working but it's really annoying since they don't mention the version number anywhere.

Know of any tricks to make this work?

Reply:
Should be handled now via https://github.com/flathub/actions/commit/4901a6d17d0dd7992e483d1d0cfa610133c62484 without anything needed here.